packages/bundle/README.md

```diff
@@ -11,3 +11,17 @@ npm install @spectrum-web-components/bundle
 
 yarn add @spectrum-web-components/bundle
 ```
+
+### Icons - Workflow
+
+While this bundle directly re-exports the majority of functionality as they would be exported from their own packages, `@spectrum-web-components/icons-workflow` is renamed to `IconsWorkflow` when leveraging the bundle. This means that you can use workflow icons in your demonstration code that using `@spectrum-web-components/bundle` like the following:
+
```

Comment:
Possible minor type, did you mean `demonstration code that uses @spectrum-web-components/bundle`?

Reply:
Good call! I found a version I liked a little bit better, but thanks for calling this out.